aten/src/ATen/native/Math.h

```diff
@@ -292,7 +292,13 @@ static inline double calc_digamma(double x) {
       // If the argument is a negative integer, NaN is returned
       return std::numeric_limits<double>::quiet_NaN();
     }
-    return calc_digamma(1 - x) - c10::pi<double> / tan(c10::pi<double> * x);
+    // Tangent function has period pi (irrational number), which is slighly different
```

Comment:
This comment is interesting, but it needs to connect to the action the code is taking; put another way, the comment should describe what the code does

In this case the code extracts the fractional part of `x`, so start with that and then explain why that's important (it's probably also important to note the units that `x` is in)